packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_services.dart

```diff
@@ -19,6 +19,7 @@ const String _packageNameKey = 'package_name';
 const String _domainsKey = 'domains';
 const String _errorCodeKey = 'errorCode';
 const String _appLinkDomainsKey = 'app_link_domains';
+const String _fingerPrintsKey = 'supplemental_sha256_cert_fingerprints';
```

Comment:
nit: _fingerprintsKey since "fingerprints" is all one word in other references to this concept